package.json

```diff
@@ -21,6 +21,9 @@
     "markdown-it-task-lists": "^2.1.1",
     "markdown-it-video": "^0.6.3",
     "prettier": "^3.0.2",
+    "markdown-link-check": "^3.11.2",
+    "markdownlint-cli2": "^0.8.1",
+    "pre-commit": "^1.2.2",
```

Comment:
i've added "significantly" to one of markdown files to trigger one fo the rules, and then run  `npm ci && npm run md-lint`   and got this error:

```
----------------------------------------------------
Spelling and writing style check
----------------------------------------------------
pre-commit-lint.sh: line 35: pre-commit: command not found
```

@ElPaisano  have no bandwidth to debug this, but I think `pre-commit` is too old (its been abandoned 7 years ago), so maybe remove it and use https://www.npmjs.com/package/husky instead?